cfme/tests/v2v/test_post_migrations.py

```diff
@@ -57,6 +61,18 @@ def test_migration_post_attribute(appliance, provider, mapping_data_vm_obj_mini,
     source_view = navigate_to(src_vm_obj, "Details")
     summary = source_view.entities.summary("Properties").get_text_of("Container")
     source_cpu, source_socket, source_core, source_memory = re.findall(r"\d+", summary)
+    snapshots_before = src_vm_obj.total_snapshots
+
+    for x in range(3):
```

Comment:
Do we really need to make 3 snapshots? Why not just 1 or 2?

Reply:
There is no hard limit of choosing it, we can not consider 1 as in case of RHV the snapshot count always be 1 after migration , so it is hard to tell if this is the expected snapshot or the old snapshot. We may take 2 but again probability of old snapshot migrated is 50-50, So i have kept it 3 to be more accurate.